next-packages/styled-base/src/index.js

```diff
@@ -158,6 +158,7 @@ let createStyled: CreateStyled = (tag: any, options?: StyledOptions) => {
               : baseTag.displayName || baseTag.name || 'Component'
           })`
 
+    Styled.defaultProps = tag.defaultProps
```

Comment:
👌

Reply:
extending defaultProps might still be an issue though, because it's quite confusing for newcomers 🤔

Reply:
This isn’t something new, it was there in create-emotion-styled, iirc it’s for when styled components get flattened. Now that I’m thinking about it, it should probably be more like this, thoughts?

```jsx
if (tag.somePropThatOnlyStyledComponentsHave !== undefined) {
  Styled.defaultProps = tag.defaultProps
}
```

Reply:
Oh, kinda thought it didnt work before. Im not sure if doing it only for emotion components is a good thing, it seems like this behaviour is useful in any case - I can't imagine when I wouldn't want those defaultProps to transition to the wrapping styled (they might be useful in function interpolations). Seems like `styled(Tag)` should ideally be completely transparent and the output component should behave pretty much the same as `<Tag/>` itself (with styling adding on top of that ofc)